pages/common/betty.md

```diff
@@ -13,7 +13,7 @@
 
```

Comment:
Nice catch. Script should also find potential help/version examples and move them down automatically if I accept it's suggestion. This will be tricky. :)